src/groovy/org/pih/warehouse/requisition/RequisitionStatus.groovy

```diff
@@ -34,7 +34,10 @@ enum RequisitionStatus {
     OPEN(0, null, StatusType.SUCCESS),
     FULFILLED(0, null, StatusType.SUCCESS),
     REVIEWING(0, null, StatusType.PRIMARY),
-    CONFIRMING(0, null, StatusType.PRIMARY)
+    CONFIRMING(0, null, StatusType.PRIMARY),
+    WAITING_FOR_APPROVAL(0, null, StatusType.WARNING),
+    APPROVED(0, null, StatusType.SUCCESS),
+    REJECTED(0, null, StatusType.DANGER)
```

Comment:
let me know what `sortOrder` those should have in your opinion - I've just added it at the end for now, to have a discussion about that.

Reply:
These new fields should be mixed in the first "section" of statuses. As you can see you added these in `// Removed` section.